xrootd/authorization_test.go

```diff
@@ -153,7 +153,7 @@ func TestWriteOriginScitokensConfig(t *testing.T) {
 	err = os.WriteFile(scitokensCfg, []byte(toMergeOutput), 0640)
 	require.NoError(t, err)
 
-	err = WriteOriginScitokensConfig()
+	err = WriteOriginScitokensConfig([]string{"/foo/bar"})
```

Comment:
Can you add a test that checks that the ScitokensConfig file looks like you except it to with this added? Or create a test that will change what the file looks like and check that it is what is expected?